pandas/core/indexes/base.py

```diff
@@ -3757,8 +3757,32 @@ def drop(self, labels, errors='raise'):
             indexer = indexer[~mask]
         return self.delete(indexer)
 
-    @Appender(base._shared_docs['unique'] % _index_doc_kwargs)
-    def unique(self):
+    base._shared_docs['index_unique'] = (
```

Comment:
I think this can be in `_index_shared_docs` instead of `base._shared_docs` ? (because `base` are the ones shared with series)